dev/com.ibm.ws.opentracing.1.x_fat/fat/src/com/ibm/ws/testing/opentracing/test/FATMPOpenTracing.java

```diff
@@ -99,22 +105,57 @@ public void testHelloWorld() throws Exception {
             tracerState += actualResponseLine;
         }
 
+        int expectedSpans = 3;
```

Comment:
why does number of expected spans change to 3 (from 2)?

Reply:
The `clearTracer()` method that was added to clear the `Tracer` object between tests.  It makes a HTTP DELETE call to clear the tracer before each test.  Therefore, each test will have an additional span from the HTTP DELETE call.